src/event.rs

```diff
@@ -533,8 +554,19 @@ pub enum WindowEvent {
     ///
     /// Platform-specific behavior:
     ///
+    /// ## iOS
```

Comment:
Could you change the `Platform-specific behavior:` to `## Platform specific` and use `### iOS` while at it?

Reply:
used `## Platform-specific` to be in line with the others